amy/extforms/forms.py

```diff
@@ -215,11 +215,10 @@ def validate_member_code(self, request: HttpRequest) -> dict:
             # find relevant membership - may raise Membership.DoesNotExist
             membership = Membership.objects.get(registration_code=member_code)
             # confirm that membership is currently active
-            if (
-                not membership.agreement_start
-                <= date.today()
-                <= membership.agreement_end
-            ):
+            # grace period: 90 days before and after
+            valid_start = membership.agreement_start - timedelta(days=90)
+            valid_end = membership.agreement_end + timedelta(days=90)
+            if not valid_start <= date.today() <= valid_end:
```

Comment:
Okay, I was commenting on this line in the previous commit which looked like:
```python
            if (
                not membership.agreement_start
                <= date.today()
                <= membership.agreement_end
            ):
```
Looks like you added a grace period - something I mentioned as possible opportunity if that code was turned into a method.

I still think this could be turned into a method.